submissions/xT0PERx/html-movie-seat-booking/main.css

```diff
@@ -0,0 +1,357 @@
+html {
+  display: grid;
+  justify-content: center;
+}
+
+.page {
+  max-width: 1550px;
+  background: rgb(201, 201, 214);
+}
+
+.top {
+  width: 100%;
+  background: linear-gradient(
+    90deg,
+    rgba(119, 109, 106, 1) 1%,
+    rgba(201, 201, 214, 1) 100%
+  );
+  border-radius: 10px;
+}
+
+.navigation {
+  margin-left: 0.5em;
+  margin-right: 0.5em;
+}
+
+.items {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 20px;
+  list-style-type: none;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+  padding-inline-start: 0em;
+}
+
+.item {
+  display: grid;
+  width: 100%;
+  padding-top: 25px;
+  padding-bottom: 25px;
+  align-self: center;
+  justify-self: center;
+  border: 10px solid;
+  border-color: transparent;
+  opacity: 0.6;
+}
+
+.item:hover {
+  border: 10px solid;
+  border-left-style: none;
+  border-right-style: none;
+  border-color: rgb(0, 0, 0);
+  border-radius: 10px;
+  cursor: pointer;
+  opacity: 1;
+}
+
+.item:active {
+  opacity: 1;
+  font-weight: 700;
+}
+
+.item a {
+  font-size: 26px;
+  color: rgb(0, 0, 0);
+  text-decoration: none;
+  text-align: center;
+}
+
+.navigation-item-choisen {
+  font-weight: 700;
+  border: 10px solid;
+  border-radius: 10px;
+  border-left-style: none;
+  border-right-style: none;
+  border-color: rgb(0, 0, 0);
+  opacity: 1;
+}
+
+.content {
+  display: grid;
+  width: 100%;
+  height: 100vh;
+  grid-template-columns: repeat(2, minmax(400px, 750px));
+  gap: 1em;
+  margin-top: 10px;
+  border-radius: 10px;
+}
+
+.placement {
+  display: grid;
+  grid-template-rows: repeat(6, [row] 1fr);
+  background-image: url(/Images/background.jpg);
+  width: 100%;
+  height: 925px;
+  padding-right: 20px;
+  background-repeat: no-repeat;
+}
+
+.screen {
+  display: grid;
+  text-align: center;
+  width: 80%;
+  padding: 15px;
+  justify-self: center;
+  align-self: center;
+  background-color: rgba(255, 255, 255, 0.6);
+  opacity: 0.8;
+  order: 3;
+}
+
+.screen p {
+  font-size: 24px;
+  font-weight: 700;
+  text-transform: uppercase;
+}
+
+.calendarWrap {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  align-items: center;
+  background-color: rgba(0, 0, 0, 0.6);
+  order: 1;
+}
+
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-column: span 5;
+  margin-block-start: 0em;
+  margin-block-end: 0em;
+  margin-inline-start: 0px;
+  margin-inline-end: 0px;
+  padding-inline-start: 0px;
+}
+
+.calendarItem {
+  display: grid;
+  justify-content: center;
```

Comment:
Simple `text-align: center;` will be enough for horizontally centering.)

Reply:
Thanks for that =)